app/src/main/java/org/oppia/app/player/exploration/ExplorationActivity.kt

```diff
@@ -22,9 +22,13 @@ private const val TAG_STOP_EXPLORATION_DIALOG = "STOP_EXPLORATION_DIALOG"
 const val TAG_HINTS_AND_SOLUTION_DIALOG = "HINTS_AND_SOLUTION_DIALOG"
 
 /** The starting point for exploration. */
-class ExplorationActivity : InjectableAppCompatActivity(), StopStatePlayingSessionListener,
+class ExplorationActivity :
+  InjectableAppCompatActivity(),
+  StopStatePlayingSessionListener,
```

Comment:
Does ktlint catch these? I am guessing it doesn't because I dont recall whether it does. If it doesn't , maybe let's consider adding a custom check for this (i.e. parameters in new lines for params that span multiple lines) Can you create an issue for this?

Reply:
Is it like if there are multiple things separated by `,` it should be all in new line?

Reply:
No it's more that if a function call is spread across multiple lines then each param should be in a single line